### ISSUE TITLE
2022.3.0 Issue with paper elements

### ISSUE THREAD

**Issue by DavidFW1960** [iantrich/podcast-card#20]
as per https://community.home-assistant.io/t/lovelace-rss-feed-parser-plugin-list-card/64637/275?u=davidfw1960
![image](https://user-images.githubusercontent.com/36390762/156478583-4f6ba7f4-97a4-4906-9a94-4ebc1c2b9e49.png)
```
              - entity: sensor.gpodder
                type: custom:podcast-card
                name: Home Assistant Podcasts
                icon: mdi:home-assistant
                default_target: media_player.office_speaker
                show_player: true
                custom_player: true
```

**Comment by DavidFW1960** [comment]
Hi Ian, just wondering if you've looked at this yet?

**Comment by iantrich** [comment]
Made some progress on switching the paper elements over to mwc over the weekend, but just didn't have the time to get it complete.

**Comment by DavidFW1960** [comment]
any further update Ian?

**Comment by iantrich** [comment]
No, sorry. I've been on vacation the past week. Will get back on Saturday and try to take a look.

**Comment by DavidFW1960** [comment]
Hi Ian, it's now almost May. are you able to fix this or should I just remove it?

**Comment by DavidFW1960** [comment]
june??

**Comment by iantrich** [comment]
Hey David, sorry for the wait. I just don't have the time to figure out what to do to fix this and am leaning towards just archiving the repo given the low userbase. Life is getting in the way and I don't see that changing anytime soon, unfortunately.
